src/main/cpp/alias/Alias.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Alias.h
+ *
+ *  Created on: Mar 11, 2017
+ *      Author: cedric
+ */
+#ifndef ALIAS_H_
+#define ALIAS_H_
```

Comment:
Can you make the include guard identifier conform to Stride's format? That would be:

```c
#ifndef ALIAS_H_INCLUDED
#define ALIAS_H_INCLUDED

...
#endif
```
```